src/plugins/score-plugin-avnd/Spat/Rotator.hpp

```diff
@@ -0,0 +1,152 @@
+#pragma once
+#include <iostream>
+//#include <vector>
+//#include <iomanip>
+#include <halp/audio.hpp>
+#include <halp/controls.hpp>
+#include <halp/meta.hpp>
+//#include <experimental/mdspan>
+
+#include <saf.h>
+//#include <rotator.h>
+//#include <cblas.h>
+
+#define MAX_ORDER 7
+#define MAX_NSH ((MAX_ORDER + 1) * (MAX_ORDER + 1))
+
+class Rotator
+{
+public:
+  halp_meta(name, "Rotator") halp_meta(c_name, "avnd_rotator")
```

Comment:
erf, i'll have to fix the .clang-format file so that it does not fuck up the indentation here...